iree/hal/vmvx/registration/driver_module_sync.c

```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/hal/vmvx/registration/driver_module_sync.h"
+
+#include <inttypes.h>
+#include <stddef.h>
+
+#include "iree/base/api.h"
+#include "iree/hal/local/executable_loader.h"
+#include "iree/hal/local/loaders/vmvx_module_loader.h"
+#include "iree/hal/local/sync_device.h"
+#include "iree/hal/local/sync_driver.h"
+#include "iree/vm/api.h"
+
+// TODO(#4298): remove this driver registration and wrapper.
+
+// TODO(benvanik): replace with C flags.
+#define IREE_HAL_VMVX_WORKER_COUNT 0
+#define IREE_HAL_MAX_VMVX_WORKER_COUNT 16
+
+#define IREE_HAL_VMVX_SYNC_DRIVER_ID 0x53564D5658lu  // SVMVX
```

Comment:
this isn't four characters like the others :O https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/google/iree%24+_DRIVER_ID%5Cs0x&patternType=regexp

I don't think there's a technical restriction on this besides needing to fit within a uint64_t though
https://github.com/google/iree/blob/7bc297e9fc8bcc7f291d84e8854a44d5161f2cd6/iree/hal/driver.h#L25-L26

Reply:
4cc (aka four-cc aka four character codes) are 4 characters :)

Reply:
I can updated it to 4cc for consistency. The uint64_t should be able to store 16-characters.

Reply:
Update the ID to SVMX :)